editor/src/components/canvas/canvas-strategies/strategies/grid-rearrange-move-strategy.spec.browser2.tsx

```diff
@@ -504,10 +504,10 @@ export var storyboard = (
         await mouseUpAtPoint(dragTarget, endPoint)
 
         {
-          const { top, left, gridColumn, gridRow } = child.style
-          expect({ top, left, gridColumn, gridRow }).toEqual({
-            gridColumn: '1',
-            gridRow: '1',
+          const { top, left, gridColumnStart, gridRowStart } = child.style
+          expect({ top, left, gridColumnStart, gridRowStart }).toEqual({
+            gridColumnStart: '1',
```

Comment:
if we do the longhand route, we should also check gridColumnEnd and gridRowEnd here (I presume they are undefined!)

Reply:
Added check for those fields to 7838f8df1.